tools/disasm/variables.txt

```diff
@@ -15954,12 +15954,6 @@
     0x80BD6880:("Bg_Haka_Tomb_InitVars","UNK_TYPE1","",0x1),
     0x80BD68A0:("D_80BD68A0","UNK_TYPE1","",0x1),
     0x80BD68A4:("D_80BD68A4","UNK_TYPE1","",0x1),
-    0x80BD6DF0:("En_Sc_Ruppe_InitVars","UNK_TYPE1","",0x1),
-    0x80BD6E10:("D_80BD6E10","UNK_TYPE1","",0x1),
-    0x80BD6E14:("D_80BD6E14","UNK_TYPE1","",0x1),
-    0x80BD6E40:("D_80BD6E40","UNK_TYPE1","",0x1),
-    0x80BD6E70:("jtbl_80BD6E70","UNK_PTR","",0x4),
-    0x80BD6E88:("D_80BD6E88","f32","",0x4),
```

Comment:
It's better not to remove these since the disassembler uses them; we may need them for other versions.